packages/nitro-protocol/test/contracts/ETHAssetHolder/deposit.test.ts

```diff
@@ -33,73 +30,66 @@ const description3 =
-          ...createDepositTransaction(destinationChannelId, '0x0', held),
         });
+        const {events} = await (await tx0).wait();
+        const depositedEvent = getDepositedEvent(events);
 
-        expect(await ETHAssetHolder.holdings(destinationChannelId)).toEqual(held);
-        await depositedEvent;
+        expect(await ETHAssetHolder.holdings(destination)).toEqual(held);
+        expect(depositedEvent).toMatchObject({
+          destination,
+          amountDeposited: bigNumberify(held),
+          destinationHoldings: bigNumberify(held),
+        });
       }
-
-      const transactionRequest = {
-        ...createDepositTransaction(destinationChannelId, expectedHeld, amount),
+      const tx = ETHAssetHolder.deposit(destination, expectedHeld, amount, {
         value: msgValue,
-      };
-      // call method in a slightly different way if expecting a revert
+      });
+
       if (reasonString) {
-        const regex = new RegExp(
-          '^' + 'VM Exception while processing transaction: revert ' + reasonString + '$',
-        );
-        await expectRevert(
-          () => sendTransaction(provider, ETHAssetHolder.address, transactionRequest),
-          regex,
-        );
+        await expectRevert(() => tx, reasonString);
       } else {
-        depositedEvent = newDepositedEvent(ETHAssetHolder, destinationChannelId);
-        const balanceBefore = await signer.getBalance();
+        // const balanceBefore = await signer.getBalance();
 
-        const tx = await signer.sendTransaction({
-          to: ETHAssetHolder.address,
-          ...transactionRequest,
+        const {events} = await (await tx).wait();
+        const event = getDepositedEvent(events);
+        expect(event).toMatchObject({
+          destination,
+          amountDeposited: heldAfter.sub(held),
+          destinationHoldings: heldAfter,
         });
-        const receipt = await tx.wait();
 
-        // catch Deposited event
-        const [eventDestination, eventAmountDeposited, eventHoldings] = await depositedEvent;
-        expect(eventDestination.toUpperCase()).toMatch(destinationChannelId.toUpperCase());
-        expect(eventAmountDeposited).toEqual(heldAfter.sub(held));
-        expect(eventHoldings).toEqual(heldAfter);
-
-        const allocatedAmount = await ETHAssetHolder.holdings(destinationChannelId);
+        const allocatedAmount = await ETHAssetHolder.holdings(destination);
         await expect(allocatedAmount).toEqual(heldAfter);
 
         // check for any partial refund
-        const gasCost = await tx.gasPrice.mul(receipt.cumulativeGasUsed);
-        await expect(
-          (await signer.getBalance()).eq(balanceBefore.sub(eventAmountDeposited).sub(gasCost)),
-        ).toBe(true);
+        // const gasCost = (await provider.getGasPrice()).mul(cumulativeGasUsed);
+        // await expect(await signer.getBalance()).toEqual(
+        //   balanceBefore.sub(event.amountDeposited).sub(gasCost),
+        // );
```

Comment:
Why comment out this check?

Reply:
When suites are run serially, each suite effectively has a lock on the signer. Now, this might be running along another test that is spending the signer's gas. The test should be rewritten to function independently of other tests.